pkg/server/start/start.go

```diff
@@ -167,15 +183,17 @@ func (o *TiltServerOptions) Config() (*apiserver.Config, error) {
 	return config, nil
 }
 
-func (o TiltServerOptions) LoopbackClientConfig() *rest.Config {
+func (o TiltServerOptions) loopbackClientConfig() *rest.Config {
 	if o.ServingOptions.BindPort == 0 {
 		panic("internal error: LoopbackClientConfig() cannot be calculated before BindPort set")
 	}
 
 	result := &rest.Config{
-		Host:  fmt.Sprintf("http://%s:%d", o.ServingOptions.BindAddress, o.ServingOptions.BindPort),
-		QPS:   50,
-		Burst: 100,
+		Host:            fmt.Sprintf("https://%s:%d", o.ServingOptions.BindAddress, o.ServingOptions.BindPort),
+		QPS:             50,
+		Burst:           100,
+		TLSClientConfig: rest.TLSClientConfig{Insecure: true},
```

Comment:
Could we actually get the CA pubkey off the options to set as `CAData`?

(It's the first value returned from `ExtraConfig.ServingInfo.Cert.CurrentCertKeyContent()`, though I think getting to config from here is kinda awkward 😬)

Reply:
ooh good call! added